src/components/Table/GridTable.tsx

```diff
@@ -195,21 +196,25 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     api.init(persistCollapse, virtuosoRef, rows);
     api.setActiveRowId(activeRowId);
     api.setActiveCellId(activeCellId);
+    api.tableState.initSortState(props.sorting, columns);
```

Comment:
Thought: We don't have it turned on for this repo (we should) but this will fail the eslint rule that `props.sorting` and `columns` are not part of the `useMemo` deps array.

So, I either led you astray by suggesting we put it in (i.e. if we really do want to call `initSortState` each time `columns` changes) _or_ (if we don't want to call `initSortState` each time `columns` changes) we could keep it here but add an in-source comment about why `props.sorting` and `columns` ar purposefully not in our `deps` array (in which case, when we eventually add that eslint rule, we'd add a `// eslint-ignore` or what not line for it)

Reply:
Yup, makes sense. I will move it back out of this `useMemo` for now.